.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.fixAll": "explicit",
+        "source.organizeImports": "explicit"
```

Comment:
Awesome! this should make it so that ci never fails for formatting

Reply:
...if you use vscode